tests/conftest.py

```diff
@@ -0,0 +1,41 @@
+from pathlib import Path
+from tempfile import TemporaryDirectory
+
+from bird_cloud_gnn.fake import generate_data
+from bird_cloud_gnn.radar_dataset import RadarDataset
+import pytest
+
+
+@pytest.fixture()
+def setup_dataset():
```

Comment:
Can you create an issue about externalizing the setup of dataset into a fixture? Because this code is duplicate of the existing test, we have to change there as well.

Reply:
I changed `setup_dataset` to `dataset_fixture` as suggested. Issue https://github.com/point-cloud-radar/bird-cloud-gnn/issues/51  about externalising the setup of data was created.